rplugin/python3/denite/source/file_rec.py

```diff
@@ -9,6 +9,29 @@
 from os import path, pardir
 from os.path import relpath, isabs, isdir, join
 from denite.util import parse_command, abspath
+import argparse
+import os
+
+def parse_command_for_scantree(cmd):
+    """Given the user choice for --ignore get the corresponding value"""
+
+    parser = argparse.ArgumentParser(description="parse scantree options")
+    parser.add_argument('--ignore', type=str, default='use_wildignore')
+
+    # the first name on the list is 'scantree.py'
+    args = parser.parse_args(cmd[1:])
+    if args.ignore == 'use_wildignore':
+        # how to get the value of vim variable?
```

Comment:
@Shougo can you help here?
How to get the value of the option wildignore from vim.

I think this is possible but I don't know how to do it.

Reply:
`self.vim.options['wildignore']`